### PR TITLE
LRSD-4522 [CP 2.0] Banner on CP

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LRSD-4522](https://liferay.atlassian.net/browse/LRSD-4522)

### CODE REVIEW

**Comment by amosfong** [comment]
Is it possible to make the whole thing one language key with the link as a {0} parameter? Then we don't have to write an exception for japanese

**Review by kylembischof — COMMENTED** [review]
can we utilize a placeholder lang key for this rather than have an if statement separate for japanese and needing multiple separate lang keys for the same banner? is this possible to do with a url?

example: "this-is-lang-key-with-x" where `x` is the place holder, and the url is injected in.

**Comment by samuelleolopes** [comment]
hi @amosfong and @kylembischof , I'm checking if this is possible.

**Comment by samuelleolopes** [comment]
@kylembischof @amosfong @ryanschuhler  I want to thank everyone for their support, I managed to implement the suggestion.

**Review by jennyhchen — APPROVED** [review]
Looks good to me. Good job Samuel! Learned something new you thought wasn't possible to do. =)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 		<p>
 			${languageUtil.get(locale, "liferays-commercial-offerings-have-been-renamed-from-lxc-to-liferay-saas-lxc-sm-to-liferay-paas-and-dxp-to-liferay-self-hosted", "Liferay's commercial offerings have been renamed from LXC to Liferay SaaS, LXC - SM to Liferay PaaS, and DXP to Liferay Self-Hosted.")}
+			${languageUtil.format(locale, "for-more-information-please-click-x",["<a href='https://help.liferay.com/hc/articles/27114210743437' target='_blank'>" + languageUtil.get(locale, "here", "here") + "</a>"])}
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html:9]: can we add a space after `"for-more-information-please-click-x",` to stay consistent with the rest of the param commas

**Review by kylembischof — COMMENTED** [review]
Is there any manual steps that need to be taken to add or update language translations to the control panel > language override section like the original ticket has listed in the release notes? https://liferay.atlassian.net/browse/LRSD-3920

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html (hunk 1 above)
Comment (jennyhchen) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html:9]: Formatting-wise...

Samuel... can you also please merge all the commits into one? It keeps the commits simple when we send it to BChan and avoid having us (1) cherry pick multiple commits and (2) possibly reduce merge conflicts as well.

**Review by ryanschuhler — APPROVED** [review]

**Comment by samuelleolopes** [comment]
> Is there any manual steps that need to be taken to add or update language translations to the control panel > language override section like the original ticket has listed in the release notes?

yes, I will add it to the ticket.

**Comment by samuelleolopes** [comment]
> Looks good to me. Good job Samuel! Learned something new you thought wasn't possible to do. =)

@jennyhchen Yes, I am happy with that. Thank you for everyone's support.

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by kylembischof** [comment]
ci:forward

**Comment by kylembischof** [comment]
ci:forward:force

**Comment by samuelleolopes** [comment]
ci:forward:force

**Comment by jpince** [comment]
@samuelleolopes @kylembischof the js-unit test failure looks unrelated, and I also see it in [another PR](https://github.com/liferay-frontend/liferay-portal/pull/4257#issuecomment-2165998564). It might be an intermittent failure. I think we should manually forward to Brian.

**Comment by samuelleolopes** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/151406

**Pull request closed without merging**